flink-libraries/flink-cep/src/main/java/org/apache/flink/cep/CEP.java

```diff
@@ -36,6 +40,14 @@
 	 * @return Resulting pattern stream
 	 */
 	public static <T> PatternStream<T> pattern(DataStream<T> input, Pattern<T, ?> pattern) {
-		return new PatternStream<>(input, pattern);
+		return new PatternStream<>(input, pattern, skipStrategy);
+	}
+
+	/**
+	 * Set the pattern's skip strategy after match.
+	 * @param afterMatchSkipStrategy the skip strategy to use.
+	 */
+	public static void setAfterMatchSkipStrategy(AfterMatchSkipStrategy afterMatchSkipStrategy) {
```

Comment:
As @dianfu said in the corresponding JIRA, the `AfterMatchSkipStrategy` should be part of the `Pattern` not `PatternStream`.